src/async_fuse/memfs/kv_engine.rs

```diff
@@ -514,6 +849,29 @@ mod test {
         assert_eq!(key.len(), 3);
     }
 
+    #[tokio::test]
+    async fn test_lock_unlock() {
+        let test_key = "TEST_LOCK_UNLOCK";
+        let client = EtcdKVEngine::new_for_local_test(vec![ETCD_ADDRESS.to_owned()])
+            .await
+            .unwrap();
+        let key: Vec<u8> = Vec::from(test_key);
+        let key = LockKeyType::FileNodeListLock(key);
+        let lock_key = client.lock(&key, Duration::from_secs(9999)).await.unwrap();
+        // start a new thread to lock the same key
+        let handle = tokio::spawn(async move {
+            let client2 = EtcdKVEngine::new_for_local_test(vec![ETCD_ADDRESS.to_owned()])
```

Comment:
Add some flag or other logic to check if lock() in task return after the outter unlock()

Reply:
The reason for doing this is what?

Reply:
To check the lock logic, otherwise this test is meaningless